### ISSUE TITLE
FiraCodeNerdFont doesn't work on Windows 10

### ISSUE THREAD

**Issue by linj121** [face-hh/griddycode#123]
**Describe the bug**

I've installed FiraCodeNerdFont but all the icons and indentations are still blurry :(

**Screenshots**

![Griddycode Screenshot 2024 04 23 - 14 02 42 22](https://github.com/face-hh/griddycode/assets/47153231/58e0120e-8b84-418a-8674-938cf84c866a)
![Griddycode Screenshot 2024 04 23 - 14 03 12 40](https://github.com/face-hh/griddycode/assets/47153231/38750163-3e5c-424b-b57c-06161de528ee)
![Griddycode Screenshot 2024 04 23 - 14 03 26 65](https://github.com/face-hh/griddycode/assets/47153231/89bbdb97-ca99-4119-95dc-7ac892d17225)

**Environment:**
 - OS: Windows 10 Home 22H2 19045.4291
 - GriddyCode Version: v1.2.1

**Comment by face-hh** [comment]
Indentation is blurry because of the way Godot handles the icons (they're built-in non-SVGs, I believe)

The icons are invalid because... reasons. We've had many people report this yet we haven't found any specific cause. Maybe downloading Godot and running the project from source could help?

I, myself, never experienced this issue on neither Linux or Windows after the version that embedded the font - and speaking of that, maybe you could try the version *before* the font picker got introduced, since that one had the font set as the entire project font.

**Comment by linj121** [comment]
> Indentation is blurry because of the way Godot handles the icons (they're built-in non-SVGs, I believe)
> 
> The icons are invalid because... reasons. We've had many people report this yet we haven't found any specific cause. Maybe downloading Godot and running the project from source could help?
> 
> I, myself, never experienced this issue on neither Linux or Windows after the version that embedded the font - and speaking of that, maybe you could try the version _before_ the font picker got introduced, since that one had the font set as the entire project font.

Yeah I'll try running it from the source then. Thanks for the quick response!

**Comment by Autonvilkku** [comment]
I discovered this on linux. I exported it myself

**Comment by rong-xiaoli** [comment]
Sorry to bother but I encountered the same issue on my computer...

![Image](https://github.com/user-attachments/assets/f03b84cf-c155-42bd-8c0d-6a20be52e884)

Don't know what happened, but I've installed FiraMono NerdFont, FiraCode NerdFont, 0xProto NerdFont, and Symbols NerdFont. 

Editor version 1.2.2
